talk/talk.qmd

```diff
@@ -266,20 +266,27 @@ Ciric et al., 2022, *Nature Methods*
 ::::
 
 
-## Middleware: SDCFlows
+## Middleware: SDCFlows (SDC = susceptibility distortion correction)
 
-- Susceptibility distortion correction is a significant source of analytical flexibility
-    - Multiple fieldmap sequences (spin echo, gradient echo, scanner-reconstructed $B_0$ maps)
-    - Multiple correction implementations with different data structures
-- Comparability among methods is poor, with little insight into sources of variation
+- **What?** Wraps or implements methods for **estimating** field maps in various scenarios:
+    - covering (i) so-called *TOPUP*, (ii) phase-difference fieldmaps and their variants, (iii) *fMRIPrep*'s "fieldmapless"; but
+    - it does not cover point-spread-function approaches (extremely marginal).
 
-- SDCFlows
-    - Defines an intermediate representation of a field inhomogeneity map
-    - Wraps or implements methods for estimating this map from different sequences
-    - Provides a single method for applying estimated maps to EPI sequences
+- **How?**
+    - Defines a shared representation model (B-Spline) for the field map (comparability ↑↑, methodological variability ↓↓)
+    - Technical perk: "decouples" estimation and application steps (modularity ↑↑, methodological variability ↑).
```

Comment:
here I say methodological variability increases because, effectively, you can try several things and see which one sticks with relative ease.